BlockServer/fileIO/file_manager.py

```diff
@@ -293,3 +293,21 @@ def get_path(name, is_component):
             path = os.path.abspath(FILEPATH_MANAGER.get_config_path(name))
 
         return path
+
+    @staticmethod
+    def get_bumpstrip_config():
+        """
+        Parses the bump strip config file into a list of BoolStr objects.
+
+        Returns:
+            XML root node of the file if it exists, empty list if it doesn't exist or fails to parse.
+        """
+        if os.path.exists(FILEPATH_MANAGER.get_banner_path()):
+            try:
+                return ConfigurationXmlConverter.banner_config_from_xml(
+                    ConfigurationFileManager._attempt_read(FILEPATH_MANAGER.get_banner_path())
+                )
+            except (IOError, ElementTree.ParseError) as ex:
+                # XML failed to parse. Log the error and return an empty list
+                print_and_log("Failed to parse banner xml file. Error was {} {}".format(ex.__class__.__name__, ex))
+        return []
```

Comment:
I'm still not entirely set on my opinion of multiple `return` statements. 

I think in this case it is less readable because I expect the standard route through the function to return at the end. As it is, your standard route through the function returns mid-way through in a non-obvious (i.e. >1s to find on first inspection) place. 

I think that's less readable than having your exception case (e.g. parsing error or file not existing) returning a special value.

Have a think about it.